example/src/main/java/com/github/appintro/example/ui/java/JavaIntro.java

```diff
@@ -0,0 +1,54 @@
+package com.github.appintro.example.ui.java;
+
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+
+import com.github.appintro.AppIntro;
+import com.github.appintro.AppIntroFragment;
+import com.github.appintro.example.R;
+
+
+public class JavaIntro extends AppIntro {
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+
+        addSlide(AppIntroFragment.newInstance("Welcome!",
+                "This is a demo of the AppIntro library, with a custom background on each slide!",
```

Comment:
Maybe this can be customized to mention that this is an example in Java?

Reply:
done